src/borg/testsuite/archiver/__init__.py

```diff
@@ -172,8 +172,9 @@ def cmd(self, *args, **kw):
         output = empty.join(line for line in output.splitlines(keepends=True) if pp_msg not in line)
         return output
 
-    def create_src_archive(self, name):
-        self.cmd(f"--repo={self.repository_location}", "create", "--compression=lz4", name, src_dir)
+    def create_src_archive(self, name, ts=None):
+        timestamp_arg = f"--timestamp={ts}" if ts else ""
```

Comment:
this is what causes the current test fails. it is because there is an empty string argument.